src/conf_mode/interfaces_bridge.py

```diff
@@ -117,7 +119,20 @@ def get_config(config=None):
 
     return bridge
 
+def verify_vxlan_neighbor_suppress(bridge):
```

Comment:
Is there a reason for this to be a dedicated function? As this is not reused, it can happily live in verify()

Reply:
Done